src/openseadragon.js

```diff
@@ -1248,6 +1248,7 @@ function OpenSeadragon( options ){
             },
             zoomPerClick:           2,
             zoomPerScroll:          1.2,
+            zoomPerDblTapDrag:      1.2,
```

Comment:
I think we should stick with `DblClick` instead of `DblTap`. All of our gestures can work with mouse, touch, or pen, and we're using the word "click" to refer to all of them.

Reply:
Also, we'll need a JSDoc comment for this new property. You can add it near the JSDoc comments for the related properties.